src/serverlist.cpp

```diff
@@ -490,6 +542,107 @@ void CServerListManager::CentralServerQueryServerList ( const CHostAddress& Inet
     }
 }
 
+int CServerListManager::IndexOf ( CHostAddress haSearchTerm )
+{
+    // Find the server in the list. The very first list entry
+    // per definition is the directory server
+    // (i.e., this server).
+    for ( int iIdx = ServerList.size() - 1; iIdx > 0; iIdx-- )
+    {
+        if ( ServerList[iIdx].HostAddr == haSearchTerm )
+        {
+            return iIdx;
+        }
+    }
+    return INVALID_INDEX;
+}
+
+void CServerListManager::CentralServerLoadServerList ( const QString strServerList )
+{
+    QFile file ( strServerList );
+
+    if ( !file.open ( QIODevice::ReadWrite | QIODevice::Text ) )
+    {
+        qWarning() << qUtf8Printable (
+            QString ( "Could not open '%1' for read/write.  Please check that Jamulus has permission (and that there is free space)." )
```

Comment:
Isn't there one space too much?

Reply:
I'm old.  Full stops get two spaces.  I can remove the annoying suggestion as well as the spaces.